lib/parsers.js

```diff
@@ -31,7 +31,7 @@ module.exports.parseError = function parseError(err, kwargs, cb) {
     var extraErrorProps;
     for (var key in err) {
       if (err.hasOwnProperty(key)) {
-        if (key !== 'name' && key !== 'message' && key !== 'stack') {
+        if (key !== 'name' && key !== 'message' && key !== 'stack' && key !== 'domain') {
```

Comment:
I personally dislike newlines in the middle of an if conditional because of how the indentation ends up looking, so I avoid them as much as possible, but the alternative I would consider is:
```javascript
if (['name', 'message', 'stack', 'domain'].indexOf(key) === -1) {
```
Arguable whether this is more readable, I could go either way.